src/input/Input.js

```diff
@@ -67,7 +67,7 @@ class Input extends React.Component {
       leftIconContainerStyle,
       rightIcon,
       rightIconContainerStyle,
-      inputComponent: InputComponent = TextInput,
+      InputComponent: InputComponent = TextInput,
```

Comment:
```suggestion
      InputComponent = TextInput,
```

Reply:
Thanks for the catch. I pushed another commit to resolve this.

Reply:
You still need to change this

Reply:
I tried that change, but the tests fail. I'm not sure why though. I thought that should provide that variable as a default? But it doesn't seem to.

Reply:
OKay I'd say put it as `InputComponent`,

Then add a default props section at the bottom.

```js
Input.defaultProps = {
  InputComponent: TextInput
}
```

Reply:
The tests also use snapshot testing. So you need to update the snapshots. See https://react-native-elements.github.io/react-native-elements/docs/testing.html#snapshot-testing

Reply:
Alright got that piece working.